src/index.css

```diff
@@ -1,5 +1,5 @@
 /* component styles */
-.hide:not(.match-in-children) {
+.rdts-hide:not(.match-in-children) {
```

Comment:
The problem I am having here is, that with using bootstrap < 4, there is already a CSS definition that hides any element with `.hide` class. And without this change, the parents are not displayed when `keepTreeOnSearch` option is used.

You can test it out here, with search for `iWay`, the parent of iWay should be still visible, but because of Bootrstrap 3, it is not
https://codesandbox.io/s/927kl26w34

Reply:
Ok, I found a solution without changing that. I needed to use another `!important` in my CSS, but that is inevitable when bootstrap is bringing out the hammer 😉